config/environments/development.rb

```diff
@@ -59,4 +60,23 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+
+  # Allow connections to local server
+  config.hosts.clear
+  config.action_mailer.delivery_method = :smtp
+  host = ENV["host"]
+  config.action_mailer.default_url_options = {
+    host: host,
+    protocol: "http"
+  }
+
+  ActionMailer::Base.smtp_settings = {
+    :address => ENV["mail_address"],
```

Comment:
dùng cú pháp này em nhé, sửa tương tự bên dưới
```suggestion
    address: ENV["mail_address"],
```